content/docs/practices/instrumentation.md

```diff
@@ -190,8 +190,18 @@ that large, investigate alternate solutions such as reducing the number of
 dimensions or moving the analysis away from monitoring and to a general-purpose
 processing system.
 
-If you are unsure, start with no labels and add more
-labels over time as concrete use cases arise.
+To give you a better idea of the underlying numbers, let's look at node_exporter.
+node_exporter exposes metrics for every mounted filesystem. Every node will have
+in the tens of timeseries for, say, `node_filesystem_avail`. If you have
+10,000 nodes, you will end up with roughly 100,000 timeseries for
+`node_filesystem_avail`, which is fine for Prometheus to handle.
+
+If you were to now add quota per user, you would quickly reach a double digit
+number of millions with 10,000 users on 10,000 nodes. This is too much for the
+current implementation of Prometheus.
```

Comment:
It's not just about what Prometheus can handle, it's the opportunity cost of all the other metrics you can't have due to how big this one is and the operational/querying cost of so much data.